crates/playground/js/Wasm.tsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import * as Comlink from "comlink";
+import { Api } from "./wasm.worker";
```

Comment:
When I read `Api` I think of backends, maybe we could call this more explicitly `WorkerApi` or so?

Reply:
Done